### DISCUSSION TITLE
Prestashop v8 BUG total price had change

### DISCUSSION THREAD

**Discussion by ichigokira** [PrestaShop/PrestaShop#36315] [Q&A]
Prestashop V8- 
BUG when editing the customer's information like address in the order, 
problem with the price had change

**Comment by kpodemski** [comment]
Hello @ichigokira 

Did you want to report a bug or ask a question? Could you be more specific? What version of PrestaShop do you use? What to do to reproduce the problem?

**Reply by ichigokira** [reply]
Each time the customer's order information changes, the product amount decreases in the PDF. 
Prestashop v8.0.4 Creates a new PDF each time information is modified.
do you have any idea how it's possible and how can i solved

**Reply by kpodemski** [reply]
I would start by upgrading your PrestaShop version to PrestaShop 8.1.6, I couldn't reproduce this on this version.

Also, there's a setting available to recalculate shipping costs on order edit, but I understand that you meant product prices.
https://capture.dropbox.com/q5xU0uEuOtMGs5l1
